pymatgen/core/structure.py

```diff
@@ -625,6 +625,35 @@ def add_spin_by_site(self, spins: Sequence[float]) -> None:
                 new_species[Species(sym, oxidation_state=oxi_state, spin=spin)] = occu
             site.species = Composition(new_species)
 
+    def add_interstitial(self, atom: str) -> None:
```

Comment:
It would probably be better to use an [`Element`](https://pymatgen.org/pymatgen.core.html#pymatgen.core.periodic_table.Element) here instead of a string.